.gitignore

```diff
@@ -22,4 +22,4 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
-!/.idea/
+.idea/
```

Comment:
Pull request is for javadoc, and thus should not change configuration files.

Reply:
You have previously given me feedback about turning off this. Due to this, I had 27 files in my first PR. So I thought I need to change this config. May be my assumption is not right..... Could you clarify this?